release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1698,12 +1692,15 @@ With this update, the CMO now properly propagates the external labels that you c
 
 * Previously, a race condition in the `NetworkManager` restart sometimes prevented DHCP resolution to successfully complete setting up the `br-ex` bridge on node boot when using OVN-Kubernetes. With this update, `NetworkManager` is no longer restarted when setting up `br-ex`, eliminating the race condition. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055433[*BZ#2055433*])
 
+* Previously, the `PtpConfigSlave` source custom resource (CR) was set to the unsupported network_transport UDPv4, causing errors on Distributed Unit (DU) nodes. This fix updates `PtpConfigSlave` source CR to use network_transport L2 rather than UDPv4. As a result, errors are no longer found on the DU nodes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2060492#[*BZ#2060492*])
```

Comment:
Should "network_transport" be "network transport"? My apologies if I didn't spot this in my first review. It is hard to know what has changed when a PR is updated.

Reply:
You could update "This fix updates `PtpConfigSlave` source CR" to "This fix updates the `PtpConfigSlave` source CR".